ga4gh/cli.py

```diff
@@ -416,8 +417,10 @@ def client_main(parser=None):
     vsParser.set_defaults(runner=VariantSearchRunner)
     addUrlArgument(vsParser)
     addVariantSearchOptions(vsParser)
+    return vsParser
+
 
-    # variantsets/search
+def addVariantSetsSearchParser(subparsers):
     vssParser = subparsers.add_parser(
```

Comment:
I think we should use `parser` here as the variable name. There's no reason to come up with different variable names now, and they don't really help readability.